CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to the NZGB Gazetteer App
+To contribute to the project certain standards are enforced and must be follow. 
+
+## Formatting
+
+Formatting is handled by `black`.
+
+[Black](https://github.com/psf/black) is an uncompromising Python code formatting tool. It takes a Python file as an input, and provides a reformatted Python file as an output, using rules that are a strict subset of PEP 8. It offers very little in the way of configuration (line length being the main exception) in order to achieve formatting consistency. It is deterministic - it will always produce the same output from the same inputs.
+
+The line length configuration is stored in pyproject.toml.
+
+## Linting
+
+Linting is handled by `pylint`.
+
+[Pylint](https://www.pylint.org/) checks Python files in order to detect syntax errors and potential bugs (unreachable code / unused variables), provide refactoring help,
+
+The configuration is stored in .pylintrc.
+
+## Commit message
+
+This repository uses Conventional Commits
```

Comment:
as I've never heard of `Conventional Commits`, it would help if a link was given to a formal document about it..